packages/hydrogen/package.json

```diff
@@ -85,8 +85,8 @@
     "vitest": "^1.0.4"
   },
   "peerDependencies": {
-    "@remix-run/react": "^2.1.0",
-    "@remix-run/server-runtime": "^2.1.0",
+    "@remix-run/react": "^2.9.2",
+    "@remix-run/server-runtime": "^2.9.2",
```

Comment:
It's curious to me that we forgot to update these 😬

Reply:
looks like these are still at 2.1.0 because they are `peerDependencies`. Reverted back to the old version so peerDependencies can stay in a wider range

Reply:
yeah let's keep the peer deps low for now